web/src/reducers/user.js

```diff
@@ -30,6 +37,8 @@ const user = (state = initialState, action) => {
       };
     case GET_USER_FAILURE:
       return { ...state, fetching: false, error: action.error };
+    case LOGOUT_SUCCESS:
+      return initialState;
```

Comment:
It occurs to me that we should probably reset the state globally on logout.

Reply:
yes we should